app/helpers/hyrax/work_form_helper.rb

```diff
@@ -97,112 +97,5 @@ def form_file_set_select_for(parent:)
         hash[presenter.title_or_label] = presenter.id
       end
     end
-
-    ##
-    # This helper retrieves errors based on form type.
```

Comment:
since we're no longer rendering error messages from "app/views/hyrax/base/_form.html.erb", these methods are no longer called.